about.html

```diff
@@ -12,53 +12,75 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  <header>
+    <nav>
+        <img src="img/lambda-black.png" alt="">
+        <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+    </nav>
+    <img src="img/jumbo-about.png" alt="about page header image">
+</header>
+
+    <div class="containerAbout">
+     
+    <h2>About</h2>
 
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
```

Comment:
Nice job using semantic HTML for the header/nav and for the content of the page with sections.